### ISSUE TITLE
CRAN checks failing

### ISSUE THREAD

**Issue by sckott** [ropensci/taxize#580]
https://cran.r-project.org/web/checks/check_results_taxize.html

may be related to `vegan` - news - https://cran.rstudio.com/web/packages/vegan/news.html

**Comment by sckott** [comment]
and Kurt asked to fix soon, so send to cran soon

**Comment by jarioksa** [comment]
Yes, this seems to be caused by vegan. This is surprising, because I checked **vegan** with `check_packages_in_dir()` and did not see any problem.

The taxonomy in `dune.taxon` was changed to follow Chase & Reveal (2009) *Bot. J. Linnean Soc.* **161**: 122-127 (2009) which has `Subclass` as the highest relevant category (all land plants are regarded as members of Class Equisetopsida of Chlorophyta and the data has only that one Class). So `rankagg` fails in tabulating `Class`: Should tabulate `Subclass` instead.

This adds dependency on `vegan >= 2.4-2`.
